drivers/input/Kconfig

```diff
@@ -34,4 +34,11 @@ source "drivers/input/Kconfig.xpt2046"
 
 endmenu # Input Drivers
 
+config INPUT_MULTITOUCH_ENABLED
+    bool "Enable multitouch"
+    default n
+    help
+      Most applications don't need multitouch.
```

Comment:
I think this help string should instead describe *what* the Kconfig does, IE that it enables multitouch support

Reply:
Agreed, I changed it.